### PR TITLE
Solution 03 bash kernel stats

### PR DESCRIPTION
**Pull request by pahomov-and**

### CODE REVIEW

File: 03-Bash/kernel_git/kernel_stats.sh [hunk 1]
```diff
@@ -26,6 +26,13 @@ count_files_in_dir(){
 	echo "count *.py :\t $(find "$1" -type f -name "*.py" | wc -l)"
 }
 
+count_comits(){
+	local tmp_dir="$(pwd)"
```
Comment (yekovalyov) [inline 03-Bash/kernel_git/kernel_stats.sh:30]: You can use pushd/popd for same purposes.

Comment (pahomov-and) [reply 03-Bash/kernel_git/kernel_stats.sh:30]: These commands don't work on my /bin/sh

File: 03-Bash/kernel_git/kernel_stats.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+help(){
+cat <<EOF
+For use: $(basename $0) [DIR]
+-h, --help - print program description and these options.
+EOF
+}
+
+parse_args(){
+	while true; do
```
Comment (yekovalyov) [inline 03-Bash/kernel_git/kernel_stats.sh]: Is it possible to remove the line with infinite loop?

Comment (pahomov-and) [reply 03-Bash/kernel_git/kernel_stats.sh]: ok

File: 03-Bash/kernel_git/kernel_stats.sh (hunk 1 above)
Comment (pahomov-and) [reply 03-Bash/kernel_git/kernel_stats.sh:30]: "pushd is a bash enhancement to the POSIX-specified Bourne Shell"

**Pull request merged**